start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
@x0range Good to know about the command line switch. This should solve the problem for CI purposes as we should be able to pass the same parameter as part of the command line script on Travis.  In general boolean arguments should default to False.  This used to be part of the Google Python style guide but I can not find the link at present.  Will try to find a reference for this.